### PR TITLE
Maintenance work

### PR DESCRIPTION
**Pull request by simolus3**
- Migrate to GitHub actions
- Replace deprecated `pub` commands with `dart pub`
- Adhere to the recommended lints in `package:lints`
- Remove the deprecated `authors` field from the pubspec
- Use `@use` over `@import` in examples
- Remove a notice about barback transformers, I don't think anyone is still using or confused about those :D
- Remove the readme template file, I'm not sure what use it had?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,41 +8,32 @@ of [sass][2].
...
+  build_runner: ^2.1.7
 ```
 
 2\. Create `web/main.scss` containing the following code:
 
 ```scss
-@import "sub";
-@import "package:bootstrap_sass/scss/variables";
+@use "sub";
+@use "package:bootstrap_sass/scss/variables" as bootstrap;
```
Comment (nshahan) [inline README.md:29]: I'm not sure if `@use "package:..."` will automatically work like `@import` with these dart package style imports. We should add tests similar to those we have for `@import "package:..."` this before we recommend `@use`.

File: README.mdtmpl [hunk 2]
```diff
@@ -1,75 +0,0 @@
-# sass_builder
```
Comment (nshahan) [inline README.mdtmpl:1]: I know this file was used to generate the readme and ensure that the code examples in the readme match the code from the example directory. I don't know what tool exactly was being used to generate the README.md file. 

@luisvt Do you remember what tool you used?

File: example/README.md [hunk 3]
```diff
@@ -5,10 +5,7 @@ will be deleted.__
 Example:
 
 ```
-$ pub run build_runner build --output out
+$ dart run build_runner build --output web:out
```
Comment (nshahan) [inline example/README.md:8]: I'm just curious, what does the 'out' -> 'web:out' change do?

File: README.md (hunk 1 above)
Comment (simolus3) [reply README.md:29]: Good point, I've added a test importing both relative and package urls through `@use`.

File: example/README.md (hunk 3 above)
Comment (simolus3) [reply example/README.md:8]: `out` emits the whole package (including readme, pubspec, `lib/` if there was any), `web:out` will just emit (generated) files in `web/` which is a more useful option for most web projects. It's also the default used by `webdev build`, so I figured we can recommend it as well here.

**Review by nshahan — APPROVED** [review]
LGTM. Regarding the question I had about the deletion of the README template file, I'm comfortable with it's removal if you are especially since you are now maintaining this package more than everyone else. 😄

**Pull request merged**